suggested_libraries.md

```diff
@@ -1,10 +1,15 @@
-# :gem: Suggested gems
+# Suggested Libraries
 
-Here is an up-to-date version of gems which we strongly suggest to include in your project.
+Here is an up-to-date version of libraries which we strongly suggest to include in your project.
 Please include them or give a good reason not to.
 
+:exclamation: Please follow the guide of each of these libraries to know how to properly install them. :exclamation:
+
+**:bulb:** Do you know all of them? Do you know why we'd like them to be included?
+
+## :gem: Suggested gems
+
 ```rb
-gem 'autoprefixer-rails'
```

Comment:
is autoprefixer set up by default with the webpacker?

Reply:
good catch. I wanted to add it to default npm libraries and then forgot. I add it

Reply:
ok...while checking that I found out that it works out of the box...

Reply:
I confirm. It uses http://cssnext.io/ by default. ❤️